### ISSUE TITLE
Allow setting udev rules from the dashboard, via special config.txt like variables

### ISSUE THREAD

**Issue by CameronDiver** [balena-io/balena-supervisor#998]
We will have variables like `BALENA_HOST_UDEV_11='some udev rule` and these should be applied by the supervisor.

This supercedes https://github.com/balena-io/balena-supervisor/issues/880.

**Comment by ZubairLK** [comment]
@CameronDiver will this work also enable us to add custom ssh keys to config.json?

**Comment by ZubairLK** [comment]
There are quite a few os.config options in config.json that we want to be able to edit.
dnsServers, ssh keys, network manager etc.

**Comment by CameronDiver** [comment]
@ZubairLK it will not, furthermore this PR needs to be rethought given the locked down nature of the environment variables we currently support. I agree it would be good to be able to change all of these values using environment variables

**Comment by ZubairLK** [comment]
@CameronDiver  yea we need a generic solution to make edits to `config.json` via supervisor.. I'd rather invest energy there instead of just udevRules. But @shaunmulligan might have a better opinion

**Comment by CameronDiver** [comment]
I fully agree on that one. Ideally I'd like some structured editor in the dashboard too, but that's a future thing.
